Utilities/Doxygen/CMakeLists.txt

```diff
@@ -109,12 +109,9 @@ mark_as_advanced(
   ITK_DOXYGEN_XML
   ITK_DOXYGEN_SERVER_BASED_SEARCH)
 
-find_package(Perl)
```

Comment:
With this change, we drop perl as a dependency for development?

Reply:
Maybe HDF5 uses it?
Modules/ThirdParty/HDF5/src/itkhdf5/CMakeLists.txt:find_package (Perl)